services/api/src/messaging/players.service.ts

```diff
@@ -146,6 +196,10 @@ const createNewPlayer = (params: {
     newPlayer.status = PlayerStatus.FigurePlaced;
   } else if (gameService.playerHasSkipped(params.roomId, params.playerId)) {
     newPlayer.status = PlayerStatus.MoveSkipped;
+  } else if (
```

Comment:
theres no `if` condition

Reply:
There is, its in the next line and wrapped with ( ). It probably got formatted this way, because the line got too large